exercises/food-chain/package.yaml

```diff
@@ -18,3 +18,4 @@ tests:
     dependencies:
       - food-chain
       - hspec
+      - hspec-expectations
```

Comment:
You know something strange... for some reason I tried importing it from `Test.Hspec.Runner`, didn't find it there, and forgot to look in `Test.Hspec`!